### PR TITLE
Update Developer Guide and User Guide

### LINKED ISSUE #298: UG DG update
Section 2.2 -> Add tables
Screenshot

### PR DESCRIPTION
**Pull request by Md-Fazil**
1. Remove resize feature for GUI and reflect the change in the developer's guide.
2. Update the User Guide to add terms to the glossary section and do minor reformatting.

Closes #298

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -584,5 +584,8 @@ Action | Format, Examples
 * **CLI**: Command Line Interface
 * **GUI**: Graphical User Interface
 * **Inventory**: List of all items stored in StoreMando
-
+* **Item**: An item is made up of an item name, location, quantity, expiry date(optional) and tags(optional). 
+* **Similar Items**: Two items are similar if they have the same spelling for name and location but differ by letter case.
```
Comment (kumsssss) [inline docs/UserGuide.md]: I think should write as Items that have the same spelling for name and location but differ by letter case.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -584,5 +584,8 @@ Action | Format, Examples
...
+* **Identical Items**: Two items are identical if they have the same name and location in terms of spelling and letter case, as well as the same expiry date.
```
Comment (kumsssss) [inline docs/UserGuide.md]: Items that have the same name and location in terms of spelling and letter case, as well as the same expiry date if it exists for both.

Comment (Md-Fazil) [reply docs/UserGuide.md]: Fixed.

File: docs/UserGuide.md (hunk 1 above)
Comment (Md-Fazil) [reply docs/UserGuide.md]: Fixed.

**Review by JayChenYJ — APPROVED** [review]
LGTM

**Pull request merged**